.ddev/config.yaml

```diff
@@ -1,15 +1,10 @@
 name: ddev-gitpod
 type: drupal9
 docroot: web
-php_version: "7.3"
+php_version: "7.4"
 webserver_type: nginx-fpm
-router_http_port: "80"
-router_https_port: "443"
-xdebug_enabled: false
-additional_hostnames: []
-additional_fqdns: []
+router_http_port: "8888"
+router_https_port: "8889"
```

Comment:
@rfay That is what confused me.

Reply:
Those are basically throwaway. We're not using the router. So we're just getting them out of the way.